examples/oneapi/cpp/source/pca/pca_dense_batch.cpp

```diff
@@ -23,8 +23,7 @@ using namespace oneapi;
 
 template <typename Method>
 void run(const dal::table& x_train, const std::string& method_name) {
-    const auto pca_desc = dal::pca::descriptor<float, Method>()
-        .set_component_count(5)
+    const auto pca_desc = dal::pca::descriptor<float, Method>(5)
```

Comment:
```suggestion
    const auto pca_desc = dal::pca::descriptor<float, Method>(/*component_count = */5)
```
I see lack of explicitness here. Maybe proposed code is better